programmers/unfinished-runner.js

```diff
@@ -0,0 +1,22 @@
+const participant = ["leo","kiki","eden","leo"];
+const completion = ["eden", "kiki"];
+function solution(participant, completion) {
+    let  counts = {};
```

Comment:
내부의 속성값들이 변하는 오브젝트, 배열 등은 const로 선언해주세요~
```suggestion
    const counts = {};
```